### PR TITLE
doubles CD timer on hivemind heal from 5 to 10 seconds

### PR DESCRIPTION
**Pull request by coldironwarrior**
## About The Pull Request

doubles CD timer on psychic heal from 5 to 10 seconds

## Why It's Good For The Game

i thought giving hivemind the same heal as queen would be better for the caste and the game versus shrike heal; after seeing how it plays for a fair bit, im not confident the frequency of the availability of the heal is quite what i'd like.

this still makes it more available and frequent than shrike heal,  but not the level of the queen heal. it's pretty good regardless of 5 or 10 seconds anyways, since you can instantly teleport to whoever you wanna heal and be there for them.

## Changelog
:cl:
balance: made hivemind's heal ability have a 10 second cooldown instead of a 5 second cooldown
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/castes/hivemind/abilities_hivemind.dm [hunk 1]
```diff
@@ -58,7 +58,12 @@
 	SEND_SIGNAL(owner, COMSIG_ESCORTING_ATOM_BEHAVIOUR_CHANGED, minions_agressive)
 	update_button_icon()
 
-/datum/action/ability/activable/xeno/psychic_cure/queen_give_heal/hivemind/can_use_action(silent = FALSE, override_flags, selecting = FALSE)
+// ***************************************
+// *********** psychic heal
+// ***************************************
+/datum/action/xeno_action/activable/psychic_cure/queen_give_heal/hivemind
+	cooldown_timer = 10 SECONDS
+/datum/action/xeno_action/activable/psychic_cure/queen_give_heal/hivemind/can_use_action(silent = FALSE, override_flags, selecting = FALSE
```
[the remark this answers is not in the document]
Comment (Lumipharon) [reply code/modules/mob/living/carbon/xenomorph/castes/hivemind/abilities_hivemind.dm:66]: wrong typepath now btw

**Review by ivanmixo — CHANGES_REQUESTED** [review]
If you're still interested at least make the checks pass

**Comment by Lumipharon** [comment]
Bruh, what is happening in this pr.
Fix the typepath
image

**Comment by coldironwarrior** [comment]
> Bruh, what is happening in this pr. Fix the typepath image

you overestimate my ability to Actually Code, and especially to Read

**Comment by coldironwarrior** [comment]
your honor, i've discovered the culprit:
#14467 it's none other than lumi himself.
i rest my case

**Comment by Lumipharon** [comment]
Actually what has happened to this pr, this is a 1 line change, and you've done all this other stuff since you opened it.

Why are you now setting the cooldown duration when checking can_use_action?

Bruh.

**Comment by coldironwarrior** [comment]
> Actually what has happened to this pr, this is a 1 line change, and you've done all this other stuff since you opened it.
> 
> Why are you now setting the cooldown duration when checking can_use_action?
> 
> Bruh.

because i didnt realize you refactorered the xeno ability proc which caused a merge conflict, which i tried to fix when it initially appeared but didnt realize why it was conflicting, which caused the horrible amalgamation of commits afterwards. if something gets merged in a file when someone else has something open in it, expect it to get messy.

i use `cooldown_duration` here because Every Other Instance of putting a xeno ability on cooldown uses the variable `cooldown_duration`. 

image
image
image
image
image
image
image


im trying to change a value of something, im not gonna go through and refactor all of the ways cooldowns on xeno abilities operate, if that's what you're asking of me.

**Pull request closed without merging**